docs/modules/ROOT/pages/how-tos/testing/integration/rerunning.adoc

```diff
@@ -1 +1,57 @@
-= Rerunning an integration test
\ No newline at end of file
+= Retriggering Integration Tests
+
+Integration test scenarios for a given snapshot can be re-triggered by adding a label to the snapshot.
+
+.Prerequisites
+- You have a snapshot that has completed all its initial tests. All initial tests must be finished before you trigger subsequent tests.
+
+- The snapshot for which tests will be retriggered has completed its initial tests.
+
+- You have CLI access to the specific OpenShift cluster. For information on obtaining CLI access, refer Getting started.
+
+- You have an up-to-date kubectl binary.  Alternatively, the `oc` binary is also compatible.
```

Comment:
This isn't _necessarily_ a requirement but I don't know what the version requirements are. This also feels like more pertinent information to have in the CLI access section than here.